data.mainnet.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't understand the difference between these files and the reference in the .env files. It seems like it should be eitheror.

Reply:
That is temporary solution we had to introduce for the MVP. We want to have the possibility to configure it per environment, hence we had to include one file per each environment. The files (so as the env variables) will be removed when we move to v2.

Reply:
Okay, so the files are placed in the repo, and then referenced via the githibstatic url?

Reply:
yeap .... it's really there just for one month so no point doing anything more... well... reasonable